projects/plugins/jetpack/extensions/blocks/anchor-fm/templates/index.js

```diff
@@ -27,49 +27,24 @@ function spotifyTemplate( { spotifyShowUrl, spotifyImageUrl } ) {
 	];
 }
 
-function podcastSection( { episodeTrack, feedUrl, coverImage } ) {
-	const { image, guid } = episodeTrack;
+function podcastSection( { episodeTrack, feedUrl } ) {
+	const { guid } = episodeTrack;
 
 	return [
-		'core/columns',
-		{
-			align: 'wide',
-		},
+		'core/group',
```

Comment:
Is there a need for the group block? Are we planning on grouping the Spotify button with the player?

Reply:
Nope, not that I know of. I'm just new to templates. 🙂 I removed the group block here: https://github.com/Automattic/jetpack/pull/18734/commits/341ce6f28008d9d0920cf017708cb910983143a4